api/environments/identities/views.py

```diff
@@ -21,6 +30,62 @@
 from util.views import SDKAPIView
 
 
+class EdgeIdentityViewSet(viewsets.ModelViewSet):
+    serializer_class = EdgeIdentitySerializer
+    permission_classes = [IsAuthenticated, NestedEnvironmentPermissions]
+    pagination_class = EdgeIdentityPagination
+    lookup_field = "identity_uuid"
+
+    @staticmethod
+    def _get_search_function_and_value(
+        search_query: str,
+    ) -> typing.Tuple[typing.Callable, str]:
+        if search_query.startswith('"') and search_query.endswith('"'):
+            return dynamo_identifier_search_functions["EQUAL"], search_query.replace(
+                '"', ""
+            )
+        else:
```

Comment:
This `else` is unnecessary. Remember that the pythonic way of doing things is to always have a 'default' return.